frontend/components/events/EventAttendees.tsx

```
@@ -0,0 +1,111 @@
+            </div>
+            <span className="text-sm text-gray-700 w-full truncate text-center" title={attendee.name}>
+              {attendee.name}
+            </span>
+          </div>
+        ))}
+      </div>
+      {hasMore && (
+        <button
+          onClick={() => setModalOpen(true)}
+          className="mt-3 text-sm font-medium text-gray-900 underline underline-offset-2 hover:text-gray-700"
+        >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)